media-block-player-master/JS/ViewControllers/filesPickerViewController.js

```diff
@@ -106,15 +106,7 @@ class FilesPickerViewController extends ViewController {
 
     presentNextController() {
         const syncFileEditViewController = new SyncFileEditViewController();
-

```

Comment:
Tato cast sa nemala vymazavat, to nie je ta povodna, toto som ja dopisoval. Vyhadzuje sa chyba v tomto mieste, ale chyba nie je sposobena tu. Objekt syncFileEditorData nie je definovany a preto by sa chyba iba posunula dalej do editora. A mena potrebujeme pri stahovani suborov.